inst/rmd/h/anchor-sections/README.md

```diff
@@ -0,0 +1,15 @@
+# Material Design Icons
+
+From: https://material.io/
+Source: https://github.com/google/material-design-icons
+
+## Icon used
+
+* `link-black-48dp.svg`
+
+## Licence
+
+Apache-2.0 - https://www.apache.org/licenses/LICENSE-2.0.html
```

Comment:
Should also mention in `LICENSE.note` and include the Author(s) in `DESCRIPTION`

Reply:
oh ok. Thanks for the advice.  I was not sure what to do for using this type of content. 

Maybe I should find another icon to use to avoid adding all that just for one icon.
Did you design it yourself for **pkgdown** ?

Reply:
Hmmm, no idea 😞 I'm sure I would've got it from a openly licensed source, but I failed to record that information.

Reply:
But given that you're only including a small single file, I think you could just put this info in the css/svg file.